proto/proto/penumbra/client/v1alpha1/client.proto

```diff
@@ -111,11 +111,19 @@ service SpecificQueryService {
   rpc ValidatorStatus(ValidatorStatusRequest) returns (ValidatorStatusResponse);
   rpc ValidatorPenalty(ValidatorPenaltyRequest) returns (ValidatorPenaltyResponse);
   rpc NextValidatorRate(NextValidatorRateRequest) returns (NextValidatorRateResponse);
+
   rpc BatchSwapOutputData(BatchSwapOutputDataRequest) returns (BatchSwapOutputDataResponse);
   rpc SwapExecution(SwapExecutionRequest) returns (SwapExecutionResponse);
+  rpc ArbExecution(ArbExecutionRequest) returns (ArbExecutionResponse);
+
+  rpc SwapExecutions(SwapExecutionsRequest) returns (stream SwapExecutionsResponse);
+  rpc ArbExecutions(ArbExecutionsResponse) returns (stream ArbExecutionsResponse);
+
   rpc LiquidityPositions(LiquidityPositionsRequest) returns (stream LiquidityPositionsResponse);
   rpc LiquidityPositionById(LiquidityPositionByIdRequest) returns (LiquidityPositionByIdResponse);
+  rpc LiquidityPositionsById(LiquidityPositionsByIdRequest) returns (stream LiquidityPositionsByIdResponse);
```

Comment:
The idea is that this would be useful for displaying the current state of a user's set of positions -- they can send a request with all of the position IDs they care about and get those specific positions back.